source_code/main_mcu/src/fido2/cborencoder.c

```diff
@@ -0,0 +1,645 @@
+/****************************************************************************
```

Comment:
does this need to be included because the encoded public key needs to be attested? :(

Reply:
yes, some of the cbor encoded data needs to be signed.